### ISSUE TITLE
[Mozilla Firefox] does not find Firefox browser and does not have search tab command

### ISSUE THREAD

**Issue by divecat76** [raycast/extensions#11920]
### Extension

https://www.raycast.com/crisboarna/mozilla-firefox

### Description

The extension does not find Firefox browser thought it is installed. The error message says 
"Error: Mozilla Firefox browser is not installed
This extension depends on Mozilla Firefox browser. You must install it to continue."

It also does not have the search tab command, only New Tab, Search History and Search Bookmarks

### Steps To Reproduce

The extension does not find Firefox browser thought it is installed. The error message says 
"Error: Mozilla Firefox browser is not installed
This extension depends on Mozilla Firefox browser. You must install it to continue."

It also does not have the search tab command, only New Tab, Search History and Search Bookmarks

**Comment by akkaky** [comment]
I have the same issue

**Comment by grandkrav** [comment]
@divecat76 @akkaky what is the output for the command below? 
```
/Applications/Firefox.app/Contents/MacOS/firefox --version
```

**Comment by akkaky** [comment]
```
$ /Applications/Firefox.app/Contents/MacOS/firefox --version
Mozilla Firefox 125.0.3
```

```
$ osascript -e 'exists application "Firefox"'
true
```

**Comment by cowmanjoe** [comment]
Same issue for me, and same output for commands 
```
$ /Applications/Firefox.app/Contents/MacOS/firefox --version
Mozilla Firefox 125.0.3
$ osascript -e 'exists application "Firefox"'
true
```

**Comment by egovelox** [comment]
You might be interested to try the [Mozeidon extension](https://www.raycast.com/egovelox/mozeidon) I developed

This extension relies on a firefox add-on and Go CLI, that output firefox tabs and bookmarks into Raycast.

If you don't mind going through the ( relatively complex ) [installation of the extension](https://github.com/egovelox/mozeidon?tab=readme-ov-file), I would be glad to have some feedback !

**Comment by Laptop765** [comment]
Navigate to `about:profiles` in a new tab and check the `Root Directory`. The [code](https://github.com/raycast/extensions/blob/main/extensions/mozilla-firefox/src/util/index.ts#L12) is expecting it to end in `.default-release`, `.default-nightly` or `.default-esr`. For whatever reason mine was something like `.default-release-581259123` so it errored out.

I was able to fix it on my machine (please take backups before trying this, I haven't tested it extensively):
1. Click the `Show in Finder` button for `Root Directory`
2. Close Firefox
3. Rename the profile folder making sure it ends in `.default-release`
4. Open `Library/Application Support/Firefox/profiles.ini` and change the text to end in `.default-release`

When I next launched Firefox everything came up as expected and now the extension works for me!

**Comment by Laptop765** [comment]
I submitted a fix (#13905) to avoid having to rename the profile folder.
